README.md

```diff
@@ -15,6 +15,31 @@ You can publish the views with:
 php artisan vendor:publish --tag=rapidez-sitemap-views
 ```
 
+## Sitemap Generation
+
+To generate the sitemap manually, use:
+```bash
+php artisan rapidez:sitemap:generate
+```
+
+If you want to schedule the sitemap generation, add the following to your application's `app/Console/Kernel.php`:
+
+```php
+protected function schedule(Schedule $schedule)
+{
+    $schedule->command('rapidez:sitemap:generate')->twiceDaily(0, 12);
+}
+```
```

Comment:
This is changed with Laravel 11, see: https://laravel.com/docs/11.x/scheduling, this is how we describe this in the Rapidez docs for the indexer: https://docs.rapidez.io/3.x/indexer.html#scheduler